AirCasting/MicrophoneSession/MicrophoneManager.swift

```diff
@@ -17,19 +17,24 @@ class MicrophoneManager: ObservableObject {
     var session: Session?
     var measurementStream: MeasurementStream?
     
+    //This variable is used to block recording more than one microphone session at a time
+    var isRecording = false
```

Comment:
Seams like this value should not be editable from outside of the class. Please add `private(set)` or `private`

Reply:
Good idea, thanks